.github/workflows/schemacode_ci.yml

```diff
@@ -99,6 +88,34 @@ jobs:
           path: coverage.xml
         if: success()
 
+  publish:
+    runs-on: ${{ matrix.os }}
+    name: Publish Python Package
+    needs: [test]
+    strategy:
+      matrix:
+        os: ["ubuntu-latest"]
+        python-version: ["3.10"]
+    steps:
+      - name: "Fetch packages"
+        uses: actions/download-artifact@v3
+        with:
+          name: dist
+          path: dist/
+      - name: "Test PyPI upload"
+        uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          user: __token__
+          password: ${{ secrets.TEST_PYPI_API_TOKEN }}
+          repository_url: https://test.pypi.org/legacy/
+          skip_existing: true
```

Comment:
This allows us to test the uploadability of every dist, not just uniquely tagged ones.

Reply:
So on each commit we test whether we could upload? won't they all appear to have the same version? Does `skip_existing` go by the version name or checksum?

Reply:
They will have the same version. It will attempt to upload, but will allow the "a package of this name already exists" failure to pass. We don't have that set for the main PyPI, so duplicate builds will still show up as an error there.